libsolidity/formal/VariableUsage.cpp

```diff
@@ -42,6 +42,15 @@ VariableUsage::VariableUsage(ASTNode const& _node)
 			if (FunctionDefinition const* funDef = SMTChecker::inlinedFunctionCallToDefinition(*funCall))
 				m_children[&n].push_back(funDef);
 		}
+		else if (auto modifierInv = dynamic_cast<ModifierInvocation const*>(&n))
+		{
+			// TODO Calls to constructors of base classes also go in here.
+			auto modifierDef = dynamic_cast<ModifierDefinition const*>(
+				modifierInv->name()->annotation().referencedDeclaration
+			);
+			if (modifierDef)
+				m_children[&n].push_back(modifierDef);
```

Comment:
"else: not supported"?

Reply:
will add on the SMTChecker side

Reply:
Actually, the SMTChecker already says that constructors are not supported, which would be reached in that case.

Reply:
Ok!